cmd/platform/deployment-template/command.go

```diff
@@ -39,8 +39,10 @@ const (
 
 // Command represents the top level deployment-template command.
 var Command = &cobra.Command{
-	Use:     "deployment-template",
-	Short:   cmdutil.AdminReqDescription("Manages deployment templates"),
+	Use: "deployment-template",
```

Comment:
I am tempted into removing these completely and moving them over just like we did with the `clusters` API ones 🤔 what do you think. Last chance to remove things easily before GA!